platform/lib/samples/Standalone.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {parse} = require('./DocumentParser.js');
+const ExampleFile = require('./ExampleFile.js');
+
+const {promisify} = require('util');
+const fs = require('fs');
+const Hogan = require('hogan.js');
```

Comment:
As far as I see we could also use nunjucks here and avoid introducing a new template engine.

Reply:
you're right, but that's something I'd prefer to introduce in another PR.